.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-  - 0.12
+  - 8
```

Comment:
might be nice to run tests on the LTS versions (4, 6 and 8)

Reply:
Added, lets see if it passes :-S